tests/parser/test_commands.py

```diff
@@ -1,32 +1,104 @@
-import itertools
-from typing import List
+from __future__ import annotations
+
+from typing import cast
 
 import pytest
 from dynamicprompts.commands import (
     Command,
     LiteralCommand,
+    SamplingMethod,
     SequenceCommand,
     VariantCommand,
     WildcardCommand,
 )
+from dynamicprompts.utils import cross
 
 from tests.consts import ONE_TWO_THREE
 
 
 @pytest.fixture
-def literals() -> List[LiteralCommand]:
-    return [LiteralCommand("hello"), LiteralCommand("world")]
+def literals() -> list[str]:
+    return ["hello", "world"]
 
 
 class TestSequence:
-    def test_length(self, literals: List[Command]):
-        sequence = SequenceCommand(literals)
+    def test_length(self, literals: list[str | Command]):
```

Comment:
The `literals` fixture always returns `list[str]` (repeated for all of these in this file)